main.js

```diff
@@ -0,0 +1,101 @@
+let beginBoard;
+let startButton = document.getElementById('start')
+let status = document.getElementById('status')
+
+startButton.addEventListener('click', function() {
+    event.target.disabled = true;
+    event.target.textContent = "Player X's turn"
+})
+
+    const humanPlayer = 'X';
+    const computerPlayer = 'O';
+
+    const winConditions = [
+        [0, 1, 2], // row
+        [3, 4, 5], // row
+        [6, 7, 8], // row
+        [0, 3, 6], // column
+        [1, 4, 7], // column
+        [2, 5, 8], // column
+        [0, 4, 8], // diagonal
+        [6, 4, 2]  // diagonal
+    ]
+
+const cells = document.querySelectorAll ('.cell'); // .cell or .board?
+startGame();
+
+function startGame() {
+    document.querySelector (".endgame").getElementsByClassName.display = "none";
+        Array.from(Array(9).keys());
+        for (let i = 0; i < cells.length; i++) {
+            cells[i].innerText = '';
+            cells[i].style.removeProperty ('background-color');
+            cells[i].addEventListener ('click', onTurn, false);
+        }
+}
+
+function onTurn(square) {
+    if (typeof beginBoard [square.target.id] === 'number') {
+        turn (square.target.id, humanPlayer)
+        if (!isTied()) turn (goodSpot(), computerPlayer);
+    }
+}
+
+function turn (squareId, player) {
+    beginBoard [squareId] = player;
+    document.getElementById(squareId).innerText = player;
+    let itsWon = isWin (beginBoard, player)
+        if (itsWon) itsOver (itsWon)
+}
+
+function isWin (board, player) {
+    let plays = board.reduce ((a, e, i) => (e === player) ? a.concat[i] : a, []);
+    let itsWon = null;
+    for (let [index, win] of winConditions.entries()) {
+        if (win.every(elem => plays.indexOf(elem) > -1)) {
+            itsWon = {
+                index: index,
+                player: player
+            };
+            break; // Is there an alternative to using 'break' with this loop?
```

Comment:
The loop will terminate on its own when it reaches the end of the array since it's a `for...of` loop. Using `break` to exit the loop based on a certian condition (e.g. a winning combination has been found) is a perfectly good use of `break`